src/index.css

```diff
@@ -4,7 +4,7 @@
 *::after {
   box-sizing: border-box;
 }
-*:focus {
+*:focus-visible {
```

Comment:
My index CSS file is significantly longer than the one in here, by about 50 lines.